examples/lights/area/script.js

```diff
@@ -3,14 +3,14 @@ import * as UTILS from '../../globals';
 const ad = UTILS.appDefaults;
```

Comment:
this example was broken btw. not sure perhaps we should remove it, the area light from three.js is an odd one.